### ISSUE TITLE
cuckoo runing issue with Vmware workstation

### ISSUE THREAD

**Issue by socengsoc** [cuckoosandbox/cuckoo#1956]
hello 
i have installed the cuckoo on oracle Vmware and i changed the default serverresult IP , but when i run the cuckoo  the error message relate to Virtualbox  has been shown  as the below 
 
////////////////////////// the error message ///////////////////
[2017-11-14 05:57:47] CuckooCriticalError: VirtualBox' VBoxManage not found at specified path "/usr/bin/VBoxManage" (as specified in virtualbox.conf). Did you properly install VirtualBox and configure Cuckoo to use it?
2017-11-14 05:57:47,694 [cuckoo] CRITICAL: CuckooCriticalError: VirtualBox' VBoxManage not found at specified path "/usr/bin/VBoxManage" (as specified in virtualbox.conf). Did you properly install VirtualBox and configure Cuckoo to use it?

**Comment by doomedraven** [comment]
if you open new issues close olds!
did you change the machinery in cuckoo.conf? from vbox to vmware?

**Comment by doomedraven** [comment]
ah wait you run vm inside of the vm?

**Comment by socengsoc** [comment]
sorry for all because this is first time i use the cuckoo , i faced many issues on installation  .
i run the vm inside windows 8.1 host not inside vm  (no nested vm ) , i get new error that related to the path of vmrun .i opened the vmware.config file  and i find line say the  path to the vmrun  i tried to list the path and l lokking for the vmrun  and not found


///////////////// the error ////////////////
2017-11-15 00:28:55] CuckooConfigurationError: The configuration value '/usr/bin/vmrun' found for vmware:vmware:path is invalid. Please update your configuration!
2017-11-15 00:28:55,277 [cuckoo] CRITICAL: CuckooConfigurationError: The configuration value '/usr/bin/vmrun' found for vmware:vmware:path is invalid. Please update your configuration!

**Comment by socengsoc** [comment]
i explore the path /usr/bin/    to search the vmrun and the result not found  .

**Comment by socengsoc** [comment]
hello 
no one  help me

**Comment by aiman-aew** [comment]
Hello Guys,

I have installed cuckoo on a Redhat machine. However I am facing the same issue. Redhat Machine is hosted on VMware and I configured the cuckoo.conf file machenery field and the vmware.conf file with no luck.

The weird thing is that I don't have the vmrun script is not found, either the VBoxManage in /usr/bin/ 

Is their any specific way to troubleshoot this?


Thank you!

@doomedraven
